models/Product.js

```diff
@@ -1,53 +1,60 @@
 const mongoose = require('mongoose');
 const Language = require('./Language').schema;
 const PrimaryImage = require('./PrimaryImage').schema;
-const Size = require('./Size').schema;
+const ImageSet = require('../modules/common/ImageSet').schema;
+const Color = require('./Color').schema;
 
 const productSchema = new mongoose.Schema({
-  category: {
+  subcategory: {
     type: mongoose.Schema.Types.ObjectId,
     ref: 'Category',
   },
   name: [Language],
   description: [Language],
+  mainMaterial: [Language],
+  innerMaterial: [Language],
+  strapLengthInCm: Number,
```

Comment:
I think we can store it `strapLength: String`. The actual value will look like "12cm" because I don't think that we are going to perform some calculations using strapLength and number will be useless here

Reply:
That would require applying the "language" structure (12cm / 12см). Also, is it "12cm" or "12 cm"? What if someone down the line wants to convert it after all? What are the benefits of going against 1NF like that?